install/helm/gloo/generate.go

```
@@ -91,11 +91,7 @@ func generateValuesYaml(version, repositoryPrefix, globalPullPolicy string) erro
 	}
 
 	// customize config as needed for dev builds
-	isReleaseVersion, err := glooVersion.IsReleaseVersion()
-	if err != nil {
-		return err
-	}
-	if !isReleaseVersion {
+	if !glooVersion.IsReleaseVersion() {
```

are we ignoring the error here?

---

the function signature changed, so it no longer returns errors

we use the linked version now instead of the version from the git context. the git commands could fail hence the old signature returned errors as well